defaults/networking-defaults.yaml

```diff
@@ -5,19 +5,31 @@ defaults:
     power_model_constant_watt: 9.550000000
```

Comment:
Per @github-benjamin-davy, I think the conventional model number here is too high. He suggests 0.0065 based on https://gauthierroussilhe.com/articles/explications-sur-l-empreinte-carbone-du-streaming-et-du-transfert-de-donnees